### ISSUE TITLE
Tabs Module: Searchfield with theme changes

### ISSUE THREAD

**Issue by s-werking** [infor-design/enterprise#7304]
**Describe the bug**
If I change the theme color, the buttons surrounding the searchfield in the header will become the selected color. 

It looks like the issue found in pull request [#7059](https://github.com/infor-design/enterprise/pull/7059) is back with the category button being a weird shape in the classic mode. 

**To Reproduce**
1. Go to http://localhost:4000/components/tabs-module/example-category-searchfield-go-button-home.html?colors=db7726&theme=classic&mode=light
2. See that the category button is a different shape and that the category button and Go button are the header theme color.
3. Change to the new mode for the theme
4. See that the category button is the correct shape but the category button and Go button are the header theme color.

**Expected behavior**
I'm not sure what should be expected in this case with a lot of theming changes lately, but I will give a screenshot of the previous version for reference.

**Version**
- ids-enterprise: v4.81.0-dev

**Screenshots**
Previous version: 
![image](https://user-images.githubusercontent.com/76002134/223854819-ac91986e-75e1-4a85-a8f9-0aa88ee98a90.png)

Current version:
![image](https://user-images.githubusercontent.com/76002134/223852580-ff33dc3a-4670-4bbb-b2a5-e2978fb1e37e.png)

**Platform**
 - Infor Application/Team Name: lawson
 - OS Version: Windows 11
 - Browser Name: chrome
 - Browser Version: 110.0.5481.178 (Official Build) (64-bit)

**Comment by tmcconechy** [comment]
https://github.com/infor-design/enterprise/issues/7297#issuecomment-1461426704 is similar...

**Comment by tjamesallen15** [comment]
Yep. Will get this ticket too @tmcconechy

**Comment by glenlieorillo** [comment]
The bug has been fixed. However, I noticed the focus on the ellipses is rectangular.


https://user-images.githubusercontent.com/102151352/227841532-8b45add8-123b-4102-8e2f-683b241e766f.mov


Unlike in the other example, it is square.

https://main-enterprise.demo.design.infor.com/components/tabs-module/example-add-tab-button.html

![image](https://user-images.githubusercontent.com/102151352/227841597-3a24e370-621a-41bc-807e-af791418e146.png)
